### PR TITLE
split netpol cases

### PR DESCRIPTION
**Pull request by changluyi**
- [ ] Make sure you have followed [Kube-OVN Code Style](https://github.com/kubeovn/kube-ovn/blob/master/CODE_STYLE.md).

#### What type of this PR
Examples of user facing changes:
- Features
- Bug fixes
- Docs
- Tests

#### Which issue(s) this PR fixes:
Fixes #(issue-number)

### CODE REVIEW

File: .github/workflows/build-x86-image.yaml [hunk 1]
```diff
@@ -362,6 +362,95 @@ jobs:
         working-directory: ${{ env.E2E_DIR }}
         run: make k8s-netpol-e2e
 
+  k8s-netpol-e2e-legacy:
+    name: Kubernetes Network Policy E2E
```
Comment (zhangzujian) [inline .github/workflows/build-x86-image.yaml:366]:
```suggestion
  k8s-netpol-legacy-e2e:
    name: Kubernetes Network Policy Legacy E2E
```

File: Makefile.e2e [hunk 2]
```diff
@@ -5,7 +5,8 @@ E2E_NETWORK_MODE := $(shell echo $${E2E_NETWORK_MODE:-overlay})
 K8S_CONFORMANCE_E2E_FOCUS = "sig-network.*Conformance" "sig-network.*Feature:NoSNAT"
 K8S_CONFORMANCE_E2E_SKIP =
 K8S_NETPOL_E2E_FOCUS = "sig-network.*Feature:NetworkPolicy"
-K8S_NETPOL_E2E_SKIP = "sig-network.*NetworkPolicyLegacy"
+K8S_NETPOL_E2E_SKIP_LEGACY = "sig-network.*NetworkPolicyLegacy"
+K8S_NETPOL_E2E_SKIP_NETPOL = "sig-network.*Netpol"
```
Comment (zhangzujian) [inline Makefile.e2e:9]:
```suggestion
K8S_NETPOL_E2E_SKIP = "sig-network.*NetworkPolicyLegacy"
K8S_NETPOL_LEGACY_E2E_FOCUS = "sig-network.*NetworkPolicyLegacy"
```

File: Makefile.e2e [hunk 3]
```diff
@@ -48,12 +49,19 @@ k8s-conformance-e2e:
 		$(call ginkgo_option,focus,$(K8S_CONFORMANCE_E2E_FOCUS)) \
 		$(call ginkgo_option,skip,$(K8S_CONFORMANCE_E2E_SKIP))
 
+.PHONY: k8s-netpol-e2e-legacy
+k8s-netpol-e2e-legacy:
+	go test ./test/e2e/k8s-network -c -o test/e2e/k8s-network/e2e.test
+	./test/e2e/k8s-network/e2e.test --ginkgo.timeout=2h \
+		$(call ginkgo_option,focus,$(K8S_NETPOL_E2E_FOCUS)) \
+		$(call ginkgo_option,skip,$(K8S_NETPOL_E2E_SKIP_NETPOL))
```
Comment (zhangzujian) [inline Makefile.e2e:57]:
```suggestion
.PHONY: k8s-netpol-legacy-e2e
k8s-netpol-legacy-e2e:
	go test ./test/e2e/k8s-network -c -o test/e2e/k8s-network/e2e.test
	./test/e2e/k8s-network/e2e.test --ginkgo.timeout=2h \
		$(call ginkgo_option,focus,$(K8S_NETPOL_E2E_FOCUS)) \
		$(call ginkgo_option,skip,$(K8S_NETPOL_E2E_SKIP))
```

File: Makefile.e2e [hunk 4]
```diff
@@ -48,12 +49,19 @@ k8s-conformance-e2e:
...
+
 .PHONY: k8s-netpol-e2e
 k8s-netpol-e2e:
 	go test ./test/e2e/k8s-network -c -o test/e2e/k8s-network/e2e.test
 	./test/e2e/k8s-network/e2e.test --ginkgo.timeout=2h \
 		$(call ginkgo_option,focus,$(K8S_NETPOL_E2E_FOCUS)) \
-		$(call ginkgo_option,skip,$(K8S_NETPOL_E2E_SKIP))
+		$(call ginkgo_option,skip,$(K8S_NETPOL_E2E_SKIP_LEGACY))
```
Comment (zhangzujian) [inline Makefile.e2e:64]:
```suggestion
		$(call ginkgo_option,skip,$(K8S_NETPOL_E2E_SKIP))
```

Comment (changluyi) [reply Makefile.e2e]: fixed all

File: .github/workflows/build-x86-image.yaml [hunk 5]
```diff
@@ -362,6 +362,95 @@ jobs:
         working-directory: ${{ env.E2E_DIR }}
         run: make k8s-netpol-e2e
 
+  k8s-netpol-legacy-e2e:
+    name: Kubernetes Network Policy E2E
```
Comment (zhangzujian) [inline .github/workflows/build-x86-image.yaml:366]:
```suggestion
    name: Kubernetes Network Policy Legacy E2E
```

Comment (changluyi) [reply .github/workflows/build-x86-image.yaml]: finished

File: .github/workflows/build-x86-image.yaml [hunk 6]
```diff
@@ -1064,6 +1153,7 @@ jobs:
       - build-centos-compile
       - k8s-conformance-e2e
       - k8s-netpol-e2e
+      - k8s-netpol-e2e-legacy
```
Comment (zhangzujian) [inline .github/workflows/build-x86-image.yaml:1156]:
```suggestion
      - k8s-netpol-legacy-e2e
```

Comment (changluyi) [reply .github/workflows/build-x86-image.yaml]: sorry for this small issue

File: .github/workflows/build-x86-image.yaml [hunk 7]
```diff
@@ -1064,6 +1153,7 @@ jobs:
       - build-centos-compile
       - k8s-conformance-e2e
       - k8s-netpol-e2e
+      - k8s-netpol-legacy-e2e
```
Comment (zhangzujian) [inline .github/workflows/build-x86-image.yaml:1156]:
```suggestion
      # - k8s-netpol-legacy-e2e
```

Some e2e cases failed, we should comment this until the failure is resolved.

Comment (changluyi) [reply .github/workflows/build-x86-image.yaml]: ok

**Review by zhangzujian — APPROVED** [review]

**Pull request merged**